docs/cmd/kn_service_migrate.md

```diff
@@ -0,0 +1,53 @@
+## kn service migrate
+
+Migrate Knative services from source cluster to destination cluster
+
+### Synopsis
+
+Migrate Knative services from source cluster to destination cluster
+
+```
+kn service migrate [flags]
+```
+
+### Examples
+
+```
+
+  # Migrate Knative services from source cluster to destination cluster by export KUBECONFIG as environment variables
+  kn migrate --source-namespace default --destination-namespace default
+
+  # Migrate Knative services from source cluster to destination cluster by set kubeconfig as parameters
+  kn migrate --source-namespace default --destination-namespace default --source-kubeconfig $HOME/.kube/config/source-cluster-config.yml --destination-kubeconfig $HOME/.kube/config/destination-cluster-config.yml
+
+  # Migrate Knative services from source cluster to destination cluster and force replace the service if exists in destination cluster
+  kn migrate --source-namespace default --destination-namespace default --force
+
+  # Migrate Knative services from source cluster to destination cluster and delete the service in source cluster
+  kn migrate --source-namespace default --destination-namespace default --force --delete
+```
+
+### Options
+
+```
+      --delete                          Delete all Knative resources after kn-migration from source cluster
+      --destination-kubeconfig string   The kubeconfig of the destination Knative resources (default is KUBECONFIG2 from ENV property)
+      --destination-namespace string    The namespace of the destination Knative resources
+      --force                           Migrate service forcefully, replaces existing service if any.
+  -h, --help                            help for migrate
+      --source-kubeconfig string        The kubeconfig of the source Knative resources (default is KUBECONFIG2 from ENV property)
+      --source-namespace string         The namespace of the source Knative resources
```

Comment:
Can we use `--namespace` as this how the other commands use the name to operate on (which as default would be the current namespace).

Reply:
Yes, changed to use `--namespace` like other commands:
https://github.com/knative/client/pull/406/commits/8f44d46bd4ebf913a9df4b225e3b792569e68a6a